### DISCUSSION TITLE
不小心删除了pmail.db,怎么办啊,在线等

### DISCUSSION THREAD

**Discussion by maryjeck** [Jinnrry/PMail#219] [Q&A]
rt

**Comment by Jinnrry** [comment]
啊。。似乎没得办呀，服务日志还在吗，日志信息里面或许能捞出部分邮件信息

**Comment by maryjeck** [comment]
问题是登录不了了

**Comment by maryjeck** [comment]
邮件倒是无所谓

**Comment by Jinnrry** [comment]
额，程序应该会自动重新建一个pmail.db，你找个sqlite数据库工具打开，或者服务器上面装个sqlite终端打开数据库，然后往user表里面插入一个用户就行

**Comment by Jinnrry** [comment]
Md5Encode(Md5Encode(password+"pmail") + "pmail2023")

密码生成规则，按这个规则加盐md5两次md5插入

**Reply by maryjeck** [reply]
好的,我试试看哈

**Reply by maryjeck** [reply]
![图片](https://github.com/user-attachments/assets/3243b703-1dc6-465c-adde-657ac7ac2ee7)
account字段是啥?

**Reply by Jinnrry** [reply]
登陆的用户名

**Reply by maryjeck** [reply]
> 登陆的用户名
谢谢大佬(●ˇ∀ˇ●)
